src/components/movieForm/MovieForm.js

```diff
@@ -1,12 +1,14 @@
-import React from "react";
+import React, {useEffect, useState} from "react";
 import "./MovieForm.scss";
 import {Dialog} from "../index";
+import {useNavigate, useParams} from "react-router-dom";

```

Comment:
it is better strategy to keep library on top and components import at the end of import list

Reply:
Done